transports/azure-storage-queues/transaction-support.md

```diff
@@ -18,9 +18,9 @@ The following [`TransportTransactionMode` levels](/transports/transactions.md) a
 
 ## Transport transaction - Receive Only
 
-The message is not removed from the queue directly after receive, but it is hidden for 30 seconds. That prevents other instances from picking it up. If the receiver fails to process the message withing that timeframe or explicitly abandons the message, then the message will become visible again. Other instances will be able to pick it up.
+Azure Storage Queues use a `Peek-Lock` model to overcome the lack of transactions. The message is not removed from the queue directly after receive, but it is hidden for 30 seconds. That prevents other instances from picking it up. If the receiver fails to process the message within that timeframe or explicitly abandons the message, then the message will become visible again. Other instances will be able to pick it up. `ReceiveOnly` provides an `at-least-once` delivery model.
```

Comment:
> `ReceiveOnly` provides an `at-least-once` delivery model.

How about

`ReceiveOnly` **[transport transaction mode](https://docs.particular.net/transports/transactions)** provides an `at-least-once` delivery model.

`ReceiveOnly` might mean nothing to a reader. And having a link is always convinient.

Reply:
good idea. Will add that.